src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -0,0 +1,476 @@
+        }
+
+        private LargeIndel( final Kryo kryo, final Input input ) {
+            super(kryo, input);
+            cigar = kryo.readObject(input, String.class);
+        }
+
+        @Override
+        protected void serialize( final Kryo kryo, final Output output ) {
+            super.serialize(kryo, output);
+            kryo.writeObject(output, cigar);
+        }
+
+        @Override
+        public int compareTo( final BreakpointEvidence that ) {
+            int result = super.compareTo(that);
+            if ( result == 0 ) result = this.cigar.compareTo(((LargeIndel)that).cigar);
+            return result;
+        }
+
+        @Override
+        public boolean equals( final Object obj ) {
+            return obj instanceof LargeIndel && compareTo((LargeIndel)obj) == 0;
+        }
+
+        @Override
+        public int hashCode() {
+            return super.hashCode() + cigar.hashCode();
+        }
+
+        @Override
+        public String toString() {
+            return super.toString() + " Indel " + cigar;
+        }
+
+        public static final class Serializer extends com.esotericsoftware.kryo.Serializer<LargeIndel> {
+            @Override
+            public void write( final Kryo kryo, final Output output, final LargeIndel largeIndel ) {
+                largeIndel.serialize(kryo, output);
+            }
+
+            @Override
+            public LargeIndel read( final Kryo kryo, final Input input, final Class<LargeIndel> klass ) {
+                return new LargeIndel(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(MateUnmapped.Serializer.class)
+    public static final class MateUnmapped extends BreakpointEvidence {
+
+        MateUnmapped( final GATKRead read, final ReadMetadata metadata ) {
+            super(read, metadata);
+        }
+
+        private MateUnmapped( final Kryo kryo, final Input input ) { super(kryo, input); }
+
+        @Override
+        public boolean equals( final Object obj ) {
+            return obj instanceof MateUnmapped && compareTo((MateUnmapped)obj) == 0;
+        }
+
+        @Override
+        public String toString() {
+            return super.toString() + " UnmappedMate";
+        }
+
+        public static final class Serializer extends com.esotericsoftware.kryo.Serializer<MateUnmapped> {
+            @Override
+            public void write( final Kryo kryo, final Output output, final MateUnmapped mateUnmapped ) {
+                mateUnmapped.serialize(kryo, output);
+            }
+
+            @Override
+            public MateUnmapped read( final Kryo kryo, final Input input, final Class<MateUnmapped> klass ) {
+                return new MateUnmapped(kryo, input);
+            }
+        }
+    }
+
+    @DefaultSerializer(InterContigPair.Serializer.class)
+    public static final class InterContigPair extends BreakpointEvidence {
+        private final int otherContigIndex;
```

Comment:
OK. The same comment would go for some of these other types of evidence -- eg for SplitRead if we could track the SA alternate alignment tags that would be useful for merging breakpoints.